OmsAgent/omsagent.py

```diff
@@ -87,7 +87,7 @@ def __str__(self):
 ProceedOnSigningVerificationFailure = True
 PackagesDirectory = 'packages'
 keysDirectory = 'keys'
-BundleFileName = 'omsagent-1.13.11-0.universal.x64.sh'
+BundleFileName = 'omsagent-1.13.40-0.universal.x64.sh'
```

Comment:
Rather than updating like this, add comment explaining it is replaced at buildtime via `apply_version.sh` and change it to placeholder like "BundleFileName = 'omsagent-0.0.0-0.universal.x64.sh'